### ISSUE TITLE
[a11y] [Loading] Component not visible in Windows High Contrast

### ISSUE THREAD

**Issue by dpersing** [Shopify/polaris-react#776]
# Issue summary

When the loading component is viewed in Windows High Contrast, the bar is not visible. This is caused by how the background colors are removed and replaced by High Contrast.

<img width="931" alt="Loading bar not visible in High Contrast" src="https://user-images.githubusercontent.com/1462085/50123715-b32b3480-0216-11e9-919a-aeacc22784d4.png">

To compare, see the progress bar, which has been styled to support High Contrast:

<img width="Progress bar component visibility in High Contrast" src="https://user-images.githubusercontent.com/1462085/50123699-a3abeb80-0216-11e9-852f-3f7c7bb96e7f.png">

## Expected behavior

When the component is viewed with High Contrast, users can see the foreground and background of the loading bar against the background of the page.

## Actual behavior

The loading bar is not visible in High Contrast, which affects users with low vision and photosensitivity.

## Steps to reproduce the problem

1. Open the loading component page in any preferred browser to see how it appears by default (`https://polaris.shopify.com/components/feedback-indicators/loading#navigation`).
1. Open Windows and turn on [High contrast mode](https://support.microsoft.com/en-us/help/13862/windows-use-high-contrast-mode). Using any of the defaults is recommended.
1. Open the loading component page in Edge.
1. Compare the page to the progress bar component example (`https://polaris.shopify.com/components/feedback-indicators/progress-bar#navigation`).

## Recommendation

Use the same `-ms-high-contrast` styles used for the progress bar in the loading bar.

## Specifications

- Are you using the React components? (Y/N): Y
- Polaris version number: 3.3.0
- Browser: Edge
- Device: Windows 10 VM on a Macbook Pro
- Operating System: Windows 10

**Comment by ruairiphackett** [comment]
@dpersing I took a look at a few combinations for `Loading` in high contrast mode:

![https://screenshot.click/2019-04-30_13-55-57.png](https://screenshot.click/2019-04-30_13-55-57.png)
```
.Loading {
  @media screen and (-ms-high-contrast: active) {
    background-color: ms-high-contrast-color('disabled-text');
  }
}

.Level {
  @media screen and (-ms-high-contrast: active) {
    background-color: ms-high-contrast-color('selected-text-background');
  }
}
```

![https://screenshot.click/2019-04-30_13-54-10.png](https://screenshot.click/2019-04-30_13-54-10.png)
```
.Loading {
  // The same styling applies in this case if you leave out the style for .Loading
  @media screen and (-ms-high-contrast: active) {
    background-color: ms-high-contrast-color('background');
  }
}

.Level {
  @media screen and (-ms-high-contrast: active) {
    background-color: ms-high-contrast-color('selected-text-background');
  }
}
```

![https://screenshot.click/2019-04-30_14-06-05.png](https://screenshot.click/2019-04-30_14-06-05.png)
```
.Loading {
  @media screen and (-ms-high-contrast: active) {
    background-color: ms-high-contrast-color('text');
  }
}

.Level {
  @media screen and (-ms-high-contrast: active) {
    background-color: ms-high-contrast-color('selected-text-background');
  }
}
```

> Use the same -ms-high-contrast styles used for the progress bar in the loading bar.

I think this is the best option out of all of the examples 👍 

I'll put in a PR for the second example above.

**Comment by dpersing** [comment]
@ruairiphackett I'm a fan of the `background`/`selected-text-background` combo too. Thank you! 👍
